File: 백준/Bronze/10798. 세로읽기/세로읽기.js
const input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');
let result = '';

for(let i=0; i<15; i++) {
    input.forEach(line => {
        const word = line[i];
        
        if(word) {
            result += word;
        };
    });
};

console.log(result);

File: 백준/Bronze/10798. 세로읽기/세로읽기.py
import sys
input = sys.stdin.readline

MAX_COUNT = 5
MAX_LENGTH = 15

result = ''
arr = []
for _ in range(MAX_COUNT):
    line = input().strip()
    arr.append(line)

for i in range(MAX_LENGTH):
    for j in range(MAX_COUNT):
        if i < len(arr[j]):
            result += arr[j][i]
    
sys.stdout.write(result)